e2e/testcafe-devextreme/tests/chat/data/index.ts

```diff
@@ -0,0 +1,67 @@
+export const lineBreaks = '\n\n';
+
+export const createUser = (id: number, name: string, url = ''): User => ({
+  id,
+  name,
+  avatarUrl: url,
+});
+
+export const timestamp = new Date(1721747399083);
+
+export const getLongText = (useLineBreaks = false, length = 1): string => {
+  const UUID = '9138cf2e-ced3-426a-bb53-4478536f690b';
+  const longItem = '1826403415222858765740359115719081097182452189907242163763639765588452014727158270738379423360950761826403415222858765740359115719081097182452189907';
+  const longItemArray = Array(length).fill(longItem);
+  const longString = longItemArray.join('');
+
+  const result = `${UUID}:${useLineBreaks ? lineBreaks : ''}${longString}`;
+
+  return result;
+};
+
+export const getShortText = (useLineBreaks = false): string => {
+  const value = `Short${useLineBreaks ? lineBreaks : ' '}text`;
+
+  return value;
+};
+
+export const generateMessages = (
+  length: number,
+  userFirst: User,
+  userSecond = userFirst,
+  useLongText = false,
```

Comment:
It's very difficult to support due to a lot of magic arguments and logic inside. It can remain as is if we don’t change these tests in the future and only do it once. Can we simplify it?

Reply:
I'm well versed in sections of this code, I don't think it would constitute a difficulty. This utility is designed to simplify message generation. You don't have to use it, you can create your own parameters as you like.